src/client.rs

```diff
@@ -1562,6 +1562,7 @@ impl LoginConfigHandler {
                 self.adapter_luid,
             ));
         n += 1;
+        msg.user_session = Config::get_option("store_usid");
```

Comment:
Config::get_option is used to store global option. store_usid is not global option, it is peer option.